lib/binding.js

```diff
@@ -1,24 +1,29 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT license.
 
-function failCheck(messages) {
+function warn(messages) {
     messages.forEach(function(message) {
         console.log('\x1b[1m\x1b[33m', message, '\x1b[0m');
     });
-    throw new Error(messages.join(' '));
+}
+
+function fail(messages) {
+    warn(messages);
```

Comment:
use red color for failing message to differentiate it with warning messages which is yellow?

Reply:
What do you think of using [npmlog](https://www.npmjs.com/package/npmlog)? It is already the dependency of Napa.js now.

Reply:
good idea!